### ISSUE TITLE
Adding "activity level" as property to activities

### ISSUE THREAD

**Issue by boweidema** [BONSAMURAIS/BONSAI-ontology-RDF-framework#22]
When the reference unit is the quantity of a flow, we need to know the activity level (e.g., annual production volume, annual extent of an environmental mechanism) in order to scale all activities to the same level, which is required to determine the balance of all flows of the same flow object, to check the global balance requirement that all outputs = all inputs within the same time period. 

So, in conclusion, we need a concept of "activity level" for each activity. Probably smartest to restrict this to "annual activity level" - at least for the time being.

**Comment by boweidema** [comment]
P.S.: For the hackathon test datasets (Exiobase and ENSO) the reference unit is 1 year, so this issue is not urgent.

**Comment by agneta20** [comment]
As I understand we would like to have all 'reference units' finally given in terms of 'per year' ('Per year in itself is already a reference unit) I think this is an additional computational stage. That would require us to develop a stage on how  other reference unit are proportional to the 'per year' based on info provided (also in the metadata).

**Comment by boweidema** [comment]
We already have the predicate proportionalTo and the ReferenceUnit which can take the value "1 year". 
So provided this information is available, any Flow can already be related to 1 year to allow for balancing.
So we do not need a separate concept of activity level.
